lib/kernel/doc/src/pg.xml

```diff
@@ -162,13 +278,22 @@
       <fsummary>Starts group membership monitoring for a scope.</fsummary>
```

Comment:
`monitor_scope()` will be incompatible once `shards_and_metadata` becomes default with this approach. The arity 1 and 2 versions needs to be compatible. An arity 3 version with an options argument where you can specify what you want to monitor would solve this.

I'm not to fond of the update lists either. The existing API returns current state, and sends messages for updates made and I think the extended monitoring should do that too.